### PR TITLE
Add new white, black, and grey colors

### PR DESCRIPTION
**Pull request by phatmann**

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -2049,6 +2049,8 @@ export class Image extends React.Component<ImageProps> {}
 export interface Colors {
   readonly primary: string;
   readonly secondary: string;
+  readonly white: string;
+  readonly bkack: string;
```
Comment (reinaldosimoes) [inline src/index.d.ts:2053]:
```suggestion
  readonly black: string;
```

**Review by reinaldosimoes — CHANGES_REQUESTED** [review]

File: src/index.d.ts (hunk 1 above)
Comment (phatmann) [reply src/index.d.ts]: Done

**Review by reinaldosimoes — APPROVED** [review]

**Comment by pranshuchittora** [comment]
Hi @phatmann any reason for this addition

**Comment by phatmann** [comment]
They were added to the JS code but the TS declarations were not updated.

**Comment by flyingcircle** [comment]
@phatmann contact me on the RNE slack for a small token thank you for this PR

**Comment by phatmann** [comment]
What’s your handle on Slack?

**Comment by flyingcircle** [comment]
@phatmann same as here.

**Pull request merged**